### ISSUE TITLE
About Enjoy_Husky_Gibson_Flagrun_ppo1

### ISSUE THREAD

**Issue by Berk035** [StanfordVL/GibsonEnv#81]
Hello everyone,

I am studying on flagrun with ppo1 algorithm nowadays. Unfortunately, I have a problem while I am trying to run enjoy_husky_gibson_flagrun_ppo1.py.
The error is:
`TypeError: __init__() missing 2 required positional arguments: 'hid_size' and 'num_hid_layers'`
I realized that, model which consists information of size and layers is lost. How can I set these parameters? 

A second question: I see a line in the same file  as `reload_name = '/home/berk/GibsonEnv/gibson/examples/train/models/flagrun_RGBD2_50.model'`. There is not any file in this location as '.model' . How can I import the model to the code and create .model file?

Thank you in advance.

**Comment by Berk035** [comment]
Up

**Comment by fxia22** [comment]
Did you use `train_husky_gibson_flagrun_ppo1.py` script to train the model? You need to run that script first. Also `hid_size` and `num_hid_layers` can be recovered from that script as well.

**Comment by Berk035** [comment]
Yes I did, but I haven't realized that the file and some trials are terminated due to size of resolution. I will try again with slight changes and inform you. 
Thank you

**Comment by Berk035** [comment]
Hello again,
I tried to apply training the husky but I am still at same point. I couldn't the create model file. When I tried to run `train_husky_gibson_flagrun_ppo1.py`, I got the error which is given followings:

![error](https://user-images.githubusercontent.com/44226026/56893913-f4690600-6a8c-11e9-82b0-e335ebfd0ddc.jpg)
![error_2](https://user-images.githubusercontent.com/44226026/56893915-f4690600-6a8c-11e9-8cbf-eed1f3efaf56.png)

I couldn't see the problem in `shuffle()`. How can I handle with this error? 
Thank you.
